src/typings.ts

```diff
@@ -15,19 +25,71 @@ export const supportedAuthnMethod = ['passwordless', 'sso', 'credentials'] as co
 export type AuthnMethod = typeof supportedAuthnMethod[number];
 
 export interface ConfigsOptions extends PrivateConfigsOptions {
+  /**
+   * Email used to pre-fill the email field in login or sign up or form.
```

Comment:
"Email address used to"?

Typically, this documentation seems to refer to email addresses not as merely "email", but as "email address" (which I'm a fan of, but I believe I've seen a few cases where it just says "email" when referring to an address).